curator/views/react/stations_today.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import $ from 'jquery'
+const date = require('../../../tools/date.js')()
+const globals = require('../../../tools/globals.js')
+const _ = require('underscore')
+
+class Stations_today extends React.Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      stations: []
+    }
+  }
+
+  componentWillMount() {
+    $.get("/stations_today", (stations) => {
+      this.setState({stations})
+    })
```

Comment:
Convert this to use the new ajax tool. If you look into it it will be easy to understand how to use it. You can also find a bunch of examples in the user app.